eng/WpfArcadeSdk/tools/TestProjects.targets

```diff
@@ -0,0 +1,26 @@
+<Project>
+  <PropertyGroup Condition="'$(WpfTest)'=='true'">
+    <RuntimeIdentifier Condition="('$(Platform)'!='AnyCPU' and '$(Platform)'!='Win32') Or $(Platform.EndsWith('64'))">win-$(Platform)</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="'$(Platform)'=='AnyCPU' or '$(Platform)'=='Win32' or '$(Platform)'=='x86'">win-x86</RuntimeIdentifier>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(WpfTest)'=='true'">
+    <TestArchitectures Condition="'$(PlatformTarget)' == '' or '$(PlatformTarget)' == 'AnyCpu'">x86</TestArchitectures>
+  </PropertyGroup>
+  
+  <ItemGroup Condition="'$(WpfTest)'=='true'">
+    <PackageReference Include="runtime.$(RuntimeIdentifier).$(MicrosoftDotNetWpfDncEngPackage)" 
+                      Version="$(MicrosoftDotNetWpfDncEngVersion)">
+      <CopyLocal>true</CopyLocal>
+      <GeneratePathProperty>true</GeneratePathProperty>
+    </PackageReference>
+
+    <None Condition="'$(RuntimeIdentifier)'=='win-x64'" 
+          Include="$(Pkgruntime_win-x64_Microsoft_DotNet_Wpf_DncEng)\lib\$(RuntimeIdentifier)\*.dll" 
```

Comment:
Might be worth a comment that these exist to ensure native dependencies are binplaced correctly with the test.